plugins/org.eclipse.fordiac.ide.application/src/org/eclipse/fordiac/ide/application/properties/InterfaceElementSection.java

```diff
@@ -161,44 +160,37 @@ private void createInstanceInfoSection(final Composite parent) {
 	}
 
 	@Override
-	public void refresh() {
-		final CommandStack commandStackBuffer = commandStack;
-		commandStack = null;
-
-		if (null != type) {
-			refreshParameterVisibility();
-			final FBNetworkElement fb = getType().getFBNetworkElement();
-			if (fb != null) {
-				infoSection.setText(
-						MessageFormat.format(Messages.InterfaceElementSection_Instance, fb.getName(), getPinName()));
-			} else { // e.g., IP address of device
-				infoSection.setText(Messages.InterfaceElementSection_InterfaceElement);
-			}
-			typeCommentText.setText(CommentHelper.getTypeComment(getType()));
+	protected void performRefresh() {
+		refreshParameterVisibility();
+		final FBNetworkElement fb = getType().getFBNetworkElement();
+		if (fb != null) {
+			infoSection.setText(
+					MessageFormat.format(Messages.InterfaceElementSection_Instance, fb.getName(), getPinName()));
+		} else { // e.g., IP address of device
+			infoSection.setText(Messages.InterfaceElementSection_InterfaceElement);
+		}
+		typeCommentText.setText(CommentHelper.getTypeComment(getType()));
 
-			configureOpenEditorButton();
+		configureOpenEditorButton();
 
-			instanceCommentText.setText(CommentHelper.getInstanceComment(getType()));
-			instanceCommentText.setForeground(getForegroundColor());
+		instanceCommentText.setText(CommentHelper.getInstanceComment(getType()));
+		instanceCommentText.setForeground(getForegroundColor());
 
-			refreshTypeInitialValue();
-			currentParameterEditor.setInterfaceElement(getType());
-			currentParameterEditor.refresh();
+		refreshTypeInitialValue();
+		currentParameterEditor.setInterfaceElement(getType());
+		currentParameterEditor.refresh();
 
-			typeText.setText(getPinTypeName());
+		typeText.setText(getPinTypeName());
 
-			connectionDisplayWidget.refreshConnectionsViewer(getType());
+		connectionDisplayWidget.refreshConnectionsViewer(getType());
 
-			if (getType() instanceof final VarDeclaration verDeclaration) {
-				currentVarConfigCheckBox.setSelection(verDeclaration.isVarConfig());
-			}
-
-			if (fb != null) {
-				setEditable(!fb.isContainedInTypedInstance());
```

Comment:
Just as double-check - this setEditable line was removed on purpose?

Reply:
unfortunately the formating here in github is bad, if you look in line 191 of the new file you'll see that it is still there.